mona_sdk/authentication.py

```diff
@@ -272,25 +276,19 @@ def _refresh_token(mona_client):
     response = _request_refresh_token_with_retries(refresh_token_key, mona_client)
     authentications_response_info = response.json()
 
-    # Log or raise an error in case one occurred.
-    # The current client token info will not change so that on the next function call
-    # the client will try to refresh the token again.
-    if not response.ok:
-        return _handle_authentications_error(
```

Comment:
did you remove this line in the case the response is not ok on purpose? shouldn't you still handle it?

Reply:
Yes I moved the bad response handle outside (to refresh_token_if_needed that calls this function) so instead of handling the bad response here, Im handling the good response here and returns the response to the caller function (instead of true/false), checks there (line 331) if response.ok=false and handle_authentication_error there. I did this cause I didn't want to pass the message/args-to-log into refresh_token (I don't think its intuitive, I feel this way is more readable. 
lmk what you think:) @nemo-mona

Reply:
that's great :-)